content/blogposts/2022/things-to-know-before-building-a-code-search-tool.md

```diff
@@ -0,0 +1,61 @@
+---
+title: '3 things to know before building a custom, in-house code search tool '
+authors:
+  - name: Quinn Slack
+    url: https://twitter.com/sqs
+publishDate: 2022-07-11T10:00-07:00
```

Comment:
I'll update this with the actual date we decide to publish this 😄

Reply:
Let's deploy this today